packages/main/test/pages/TimePickerClock.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+	<title>TimePickerClock test page</title>
+
+	<script>
+		// delete Document.prototype.adoptedStyleSheets;
+	</script>
+
+
+	<script src="../../bundle.esm.js" type="module"></script>
+
+
+	<script data-id="sap-ui-config" type="application/json">
+		{
+			"rtl": false,
+			"formatSettings": {
+                "firstDayOfWeek": 0
+            }
+		}
+	</script>
+
+	<link rel="stylesheet" type="text/css" href="./styles/TimePickerClock.css">
+
+</head>
+<body class="stepinput1auto">
+
+	<h3>TimePickerClock</h3>
+	<div>
+		<div class="clock-wrapper">
+			<ui5-time-picker-clock id="myHours12" label="myHours12" item-min="1" item-max="12" display-step="1" last-item-replacement1="0"></ui5-time-picker-clock>
+		</div>
+		<div class="clock-wrapper">
+			<ui5-time-picker-clock id="myHours00" label="myHours00" item-min="1" item-max="12" display-step="1" inner-items prepend-zero last-item-replacement="0"></ui5-time-picker-clock>
+		</div>
+		<div class="clock-wrapper">
+			<ui5-time-picker-clock id="myHours24" label="myHours24" item-min="1" item-max="12" display-step="1" inner-items prepend-zero support2400 last-item-replacement="0"></ui5-time-picker-clock>
+		</div>
+		<div class="clock-wrapper">
+			<ui5-time-picker-clock id="myMinutes" label="myMinutes" item-min="1" item-max="60" last-item-replacement="0"></ui5-time-picker-clock>
+		</div>
+		<div class="clock-wrapper">
+			<ui5-time-picker-clock id="myMinutes10" label="myMinutes10" item-min="1" item-max="60" display-step="10" value-step="10" last-item-replacement="0"></ui5-time-picker-clock>
+		</div>
+	</div>
+
+	<div>
+		<h3>Last Changed Clock - ID, value and string value</h3>
+		<ui5-input id="clockChangeId"></ui5-input>
+		<ui5-input id="clockChangeValue"></ui5-input>
+		<ui5-input id="clockChangeStringValue"></ui5-input>
+	</div>
+
+	<script>
+
+		myHours12.addEventListener('change', displayChange);
```

Comment:
In test pages we use the noConflict event names - in your case ui5-change

Reply:
Yes, you're right.